tuxemon/map_loader.py

```diff
@@ -66,23 +67,46 @@ def load_events(self, path: str) -> Iterator[EventObject]:
             path: Path to the file.
 
         """
+        yaml_data: dict[str, dict[str, dict[str, Any]]] = {}
+
         with open(path) as fp:
             yaml_data = yaml.load(fp.read(), Loader=yaml.SafeLoader)
 
         for name, event_data in yaml_data["events"].items():
+            _id = uuid.uuid4().int
             conds = []
             acts = []
-            x = event_data.get("x")
-            y = event_data.get("y")
-            w = event_data.get("width")
-            h = event_data.get("height")
+            x = (
```

Comment:
seems like you could write this as `event_data.get("x", 0)`?

Reply:
you're right!